### PR TITLE
[WINDUP-3270] - Upgrade MTA Web to use EAP 7.4

### PR DESCRIPTION
**Pull request by carlosthe19916**
https://issues.redhat.com/browse/WINDUP-3270
- Upgraded Keycloak and Resteasy version.
- Resteasy version taken from https://access.redhat.com/articles/112673#EAP_7

### CODE REVIEW

File: src/main/resources/windup-realm/windup-realm.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "mta",
   "realm": "mta",
-  "displayName": "Migration Toolkit for Applications Web Console",
+  "displayName": "Tackle Analysis Web Console",
```
Comment (mrizzi) [inline src/main/resources/windup-realm/windup-realm.json]: This shouldn't change for the EAP 7.4 upgrade, right?

File: src/main/resources/windup-realm/windup-realm.json [hunk 2]
```diff
@@ -779,7 +779,7 @@
     {
       "id": "739a78cd-ab8d-427a-93f7-4af38f0eab31",
       "clientId": "mta-web",
-      "name": "Migration Toolkit for Applications Web Console",
+      "name": "Tackle Analysis Web Console",
```
Comment (mrizzi) [inline src/main/resources/windup-realm/windup-realm.json]: Just like above.

**Review by mrizzi — COMMENTED** [review]
I don't think the EAP 7.4 should be related to branding changes. Am I right?

File: src/main/resources/windup-realm/windup-realm.json (hunk 2 above)
Comment (carlosthe19916) [reply src/main/resources/windup-realm/windup-realm.json]: Thanks @mrizzi . You're right no branding changes should be done.

**Review by mrizzi — APPROVED** [review]
LGTM, thanks @carlosthe19916 :+1:

**Pull request merged**